README.md

```diff
@@ -34,6 +34,41 @@ to know more about what happens under the hood in **OpenWISP 2**.
 
 For more information see also [Applying the Unix Philosophy to Django projects: a report from the real world](https://www.slideshare.net/FedericoCapoano/applying-the-unix-philosophy-to-django-projects-a-report-from-the-real-world).
 
+System Requirements
+===================
+
+The following specifications will run a new, *empty* instance of OpenWISP.
+Please ensure you account for the amount of disk space your use case will require, e.g. allocate
+enough space for users to upload floor plan images.
+
+Hardware (Minimum, excluding uploaded data)
+-------------------------------------------
+- 1GB Memory
+- 300MB free disk space
```

Comment:
Is 300 MB of disk space really enough to be able to install OpenWISP with the ansible playbook? If the installation gets updated by running the playbook again, additional disk space might be used by package caches.
I just want to avoid that users run into trouble easily. And a couple of GB of disk space is not much these days.

Reply:
Let's just say 2GB free space to account for any future deps?